includes/RevisionXmlBuilder.php

```diff
@@ -51,13 +51,13 @@ public function getPageMetadataByRevId( int $revId ): string {
 		}
 
 		$output = [
-			'domain_id' => $row->domain_id,
+			'domain_id' => (int)$row->domain_id,
```

Comment:
The domain ID is hexadecimal, and so needs to be string.

Reply:
fixed